packages/react-core/src/components/Brand/__tests__/Brand.test.tsx

```diff
@@ -3,34 +3,75 @@ import { render, screen } from '@testing-library/react';
 import { Brand } from '../Brand';
 import '@testing-library/jest-dom';
 
-describe('Brand', () => {
-  test('simple brand', () => {
-    const { asFragment } = render(<Brand alt="brand" />);
-    expect(asFragment()).toMatchSnapshot();
-  });
-
-  test('passing children creates picture brand', () => {
-    render(
-      <Brand alt="brand">
-        <div>test</div>
-      </Brand>
-    );
-    expect(screen.getByText('test')).toBeInTheDocument();
-  });
-
-  test('styles get spread when there are children', () => {
-    render(
-      <Brand alt="brand" data-testid="brand" style={{color: "blue"}}>
-        <div>test width</div>
-      </Brand>
-    );
-    expect(screen.getByTestId('brand')).toHaveStyle(`color: blue`);
-  });
-
-  test('styles get spread when there are no children', () => {
-    render(
-      <Brand alt="brand no children" style={{width: "30px"}} />
-    );
-    expect(screen.getByAltText('brand no children')).toHaveStyle(`width: 30px`);
-  });
+test('simple brand', () => {
+  render(<Brand alt="brand" />);
+  expect(screen.getByAltText('brand')).toBeInTheDocument();
+});
+
+test('Renders with custom class name when className prop is provided', () => {
+  render(<Brand alt="brand" className="custom-class" />);
+  expect(screen.getByAltText('brand')).toHaveClass('custom-class');
+});
+
+test('Renders passed children', () => {
+  render(
+    <Brand alt="brand">
+      <div>test</div>
+    </Brand>
+  );
+  expect(screen.getByText('test')).toBeInTheDocument();
+});
+
+test('Renders as picture when children are present', () => {
+  render(
+    <Brand alt="brand">
+      <div>test</div>
+    </Brand>
+  );
+  expect(screen.getByText('test')?.parentElement?.tagName).toBe('PICTURE');
+});
+
+test('Renders as img when no children are present', () => {
+  render(<Brand alt="brand" />);
+  expect(screen.getByAltText('brand')?.tagName).toBe('IMG');
+});
+
+test('Renders with passed src prop', () => {
+  render(<Brand alt="brand" src="test.png" />);
+  const image = screen.getByRole('img') as HTMLImageElement;
+  expect(image.src).toMatch('test.png');
+});
```

Comment:
Can we do the same test for when children are passed, just checking the src is passed to the`img` element that's automatically rendered inside the `picture` element.

Probably worth also testing the `alt` attribute gets passed when the Brand is rendered as a picture